testbed/addons/karpenter/construct.ts

```diff
@@ -0,0 +1,61 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import iam = require('@aws-cdk/aws-iam');
+
+export interface KarpenterProps {
+    cluster: eks.Cluster;
+    namespace: string;
+}
+
+export class KarpenterController extends cdk.Construct {
```

Comment:
This is sweet! I wonder how much effort this would be to toss this into `github.com/awslabs/karpenter/cdk`. Maybe a bad idea, since it's aws specific. We should do this once we separate the AWS cloud provider.